Your Meeting Buddy/Gmeet_buddy.py

```diff
@@ -0,0 +1,90 @@
+#Importing Required modules and Libraries 
+
+from selenium import webdriver
+import schedule,time
+import keyboard
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium.webdriver.chrome.options import Options
+
+#joining Credentials
+
+username="email@gmail.com"          #enter your mail id
+password="This is my password"      #enter your password
+
```

Comment:
@Yuvraj-kadale instead of harcoding the credential, take the input from the user